client/src/services/stardust/stardustTangleCacheService.ts

```diff
@@ -258,22 +259,27 @@ export class StardustTangleCacheService extends TangleCacheService {
      public async nftDetails(
         request: INftDetailsRequest,
         skipCache: boolean = false
-    ): Promise<INftOutputsResponse | undefined> {
+    ): Promise<INftDetailsResponse | undefined> {
         const cacheEntry = this._stardustSearchCache[request.network][`${request.nftId}--nft-outputs`];
 
         if (!cacheEntry?.data?.outputs || skipCache) {
             const apiClient = ServiceFactory.get<StardustApiClient>(`api-client-${STARDUST}`);
 
-            const nftDetails = await apiClient.nftDetails(request);
+            const response = await apiClient.nftDetails(request);
             this._stardustSearchCache[request.network][`${request.nftId}--nft-outputs`] = {
-                data: { outputs: nftDetails.outputs },
+                data: {
```

Comment:
This whole block can be simplified with `... = { data: { nftDetails: response } }`, so you don't have to map the fields manually as they are identical.

Reply:
Done